Dockerfile

```diff
@@ -58,9 +62,7 @@ RUN echo "ServerName localhost" | tee /etc/apache2/conf-available/fqdn.conf \
         alias
 
 COPY --chown=${USER}:${GROUP} . /var/www/vhosts/${VHOST}/private/chsdi
-WORKDIR /var/www/vhosts/${VHOST}/private/chsdi
 
-RUN pip3 install -e .
```

Comment:
Currently how the project is setup, this is probably still needed, and I would not remove this. To be checked if the pipenv editable mode (https://pipenv-fork.readthedocs.io/en/latest/basics.html#editable-dependencies-e-g-e) would do the same job, otherwise we still have the possibility to use pip3 for this task (pipenv is just a wrapper around pip)